### PR TITLE
Current financial information of a project or milestone as "actuals"

### PR DESCRIPTION
**Pull request by adognameddog**
Actuals reflect the current financial state of a project or milestone. It distincts between revenue, costs and profit. This will not be released for now, but we need that data currently.

### CODE REVIEW

**Review by ArnaudWeyts — DISMISSED** [review]

**Review by nickwaelkens — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Comment by ketlane** [comment]
To me it's not very clear what we mean by "actuals" and "actuals.result" when reading the docs, but I understand we already agreed on this naming. Can we explain it in English words (maybe an example as well) in the documentation, though?

Maybe also add a note that they are only available to admins, so that integrators will not be puzzled when they can't access the information as regular users?

**Review by ketlane — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -2713,11 +2713,15 @@ Get a list of projects.
                 + source (object)
                     + type: `deal` (string)
                     + id: `5023d7c2-80d7-4d4b-b2bd-0fcaa6a1f069` (string)
+                + actuals (object)
```
Comment (jenssegers) [inline apiary.apib]: Should we add a comment next to this saying that it is only available for admins? I know it's in the description above, but would be nice to have it here as well 👍

**Review by jenssegers — DISMISSED** [review]
Would just add a comment saying the property is only available for admins in the response documentation.

**Review by ketlane — DISMISSED** [review]

**Review by jenssegers — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Review by ArnaudWeyts — DISMISSED** [review]

**Review by nickwaelkens — DISMISSED** [review]

**Comment by stivni** [comment]
@adognameddog I would recommend making an "integration"-branch for these changes, so we can merge this towards that branch...

File: src/07-projects/projects.apib [hunk 2]
```diff
@@ -114,7 +118,10 @@ Get details for a single project.
                         + Members
                             + decision_maker
                             + member
-
+            + actuals (object) - Only available for account administrators and users with full admin access for projects.
```
Comment (stivni) [inline src/07-projects/projects.apib:121]: Q: if only certain people can see this, will this be `null` in the other case or will the property be optionally returned? That choice should be documented...

**Review by stivni — COMMENTED** [review]
also merge toward the (unexisting) integration branch?

Q : is this already in the live version? or only with a certain featureflag or include parameter?

**Review by stivni — COMMENTED** [review]
@adognameddog if you can answer/fix my question, I think we should release this

**Review by ArnaudWeyts — APPROVED** [review]

**Comment by adognameddog** [comment]
I think an integration branch is no longer necessary 🙈

**Review by mikeverf — APPROVED** [review]

**Comment by adognameddog** [comment]
The documentation of this actually (pun very much intended) changed. I'll create a new PR.

**Pull request closed without merging**